e2e/api/backend_test.go

```diff
@@ -94,22 +94,26 @@ func (s *APIBackendTestSuite) TestRaceConditions() {
 			log.Info("CreateAccount()")
 			address, pubKey, mnemonic, err := s.Backend.AccountManager().CreateAccount("password")
 			s.T().Logf("CreateAccount(), error: %v (address: %v, pubKey: %v, mnemonic: %v)", err, address, pubKey, mnemonic)
-			if err == nil {
-				// SelectAccount
-				log.Info("CreateAccount()")
-				err = s.Backend.AccountManager().SelectAccount(address, "password")
-				s.T().Logf("SelectAccount(%v, %v), error: %v", address, "password", err)
-
-				// CreateChildAccount
-				log.Info("CreateChildAccount()")
-				address, pubKey, err := s.Backend.AccountManager().CreateChildAccount(address, "password")
-				s.T().Logf("CreateAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
-
-				// RecoverAccount
-				log.Info("RecoverAccount()")
-				address, pubKey, err = s.Backend.AccountManager().RecoverAccount("password", mnemonic)
-				s.T().Logf("RecoverAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
+			if err != nil {
+				progress <- struct{}{}
+				return
 			}
+
+			// SelectAccount
+			log.Info("CreateAccount()")
+			err = s.Backend.AccountManager().SelectAccount(address, "password")
+			s.T().Logf("SelectAccount(%v, %v), error: %v", address, "password", err)
+
+			// CreateChildAccount
+			log.Info("CreateChildAccount()")
+			address, pubKey, err = s.Backend.AccountManager().CreateChildAccount(address, "password")
+			s.T().Logf("CreateAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
+
+			// RecoverAccount
+			log.Info("RecoverAccount()")
+			address, pubKey, err = s.Backend.AccountManager().RecoverAccount("password", mnemonic)
+			s.T().Logf("RecoverAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
+
```

Comment:
Could you elaborate?

Reply:
Yes, it helps me to avoid 3th err shadowing using one err vaiable.